### PR TITLE
Documentation: Improve CNI configuration section

### PR DESCRIPTION
**Pull request by tgraf**
Signed-off-by: Thomas Graf <thomas@cilium.io>

### CODE REVIEW

File: Documentation/admin.rst [hunk 1]
```diff
@@ -240,13 +240,57 @@ If you are using systemd to manage the kubelet, another option is to add a
...
+
+The CNI auto installation is performed as follows:
+
+1. The ``/etc/cni/net.d`` and ``/opt/cni/bin`` directories are mounted from the
+   host filesystem into the pod where Cilium is running.
+
+2. The file ``/etc/cni/net.d/10-cilium.conf`` is written in case it is not
+   already provided by the user.
+
+3. The binary ``cilium-cni`` is installed to ``/opt/cni/bin``. And eventual already
```
Comment (aanm) [inline Documentation/admin.rst]: `s/And/An`

Comment (tgraf) [reply Documentation/admin.rst]: Fixed

**Review by aanm — APPROVED** [review]

File: Documentation/admin.rst [hunk 2]
```diff
@@ -259,18 +303,7 @@ configuration manually.
...
-plugin:
-
-.. code:: bash
-
-    sudo mkdir -p /opt/cni
-    wget https://storage.googleapis.com/kubernetes-release/network-plugins/cni-0799f5732f2a11b329d9e3d51b9c8f2e3759f2ff.tar.gz
-    sudo tar -xvf cni-0799f5732f2a11b329d9e3d51b9c8f2e3759f2ff.tar.gz -C /opt/cni
-    rm cni-0799f5732f2a11b329d9e3d51b9c8f2e3759f2ff.tar.gz
-
+already exists on a worker node and only create it if it does not exist yet.
```
Comment (ianvernon) [inline Documentation/admin.rst]: "only creates it if it..."

**Review by ianvernon — APPROVED** [review]
LGTM apart from one minor grammatical error.

**Pull request merged**